### ISSUE TITLE
Node Module dependency issues

### ISSUE THREAD

**Issue by nickmagruder** [CouncilDataProject/cdp-frontend#111]
### Describe the Bug

Npm install failing, "unable to resolve dependency tree" for the following libraries:

Webpack, webpack-cli, testing-library/react-hooks

### Reproduction

Simply running an npm install on a new cloned local branch

### Expected Behavior

A normal, successful npm install

### Screenshots
![image](https://user-images.githubusercontent.com/71102316/141597251-332aaf54-13b9-4e1d-8c8d-6554668e9ed1.png)
![image](https://user-images.githubusercontent.com/71102316/141597291-6f4cb7ac-91ac-4efe-b221-baad1b475c38.png)

### Environment

_Any additional information about your environment._

-   Windows 10, WSL Ubuntu 20.04.2 LTS (GNU/Linux 4.19.128-microsoft-standard x86)
-   Browser Version: N/A
-   CDP Frontend Version: Unknown, latest as of 11/12/21

**Comment by nickmagruder** [comment]
I was able to get ```npm install``` to run successfully, not not ```npm run start``` or ```npm run storybook```. I updated a few of the dependencies to newer versions until everything installed successfully, my package.json versions are below.

Here are the errors I'm encountering, wondering if anyone has any ideas:

- When running ```npm run start```
![image](https://user-images.githubusercontent.com/71102316/143980614-9827d817-6a2b-49bf-8e38-d079602cea97.png)

- When running ```npm run storybook``` (just a portion of the error)
![image](https://user-images.githubusercontent.com/71102316/143980742-a7485b3a-f960-4d3d-9d33-c18e684cb759.png)
![image](https://user-images.githubusercontent.com/71102316/143980790-c9b62657-8c62-438c-ad0d-b407659b9281.png)
![image](https://user-images.githubusercontent.com/71102316/143980899-066872f9-0611-4319-b75c-b137135cfb29.png)






package.json:
```
  "devDependencies": {
    "@babel/cli": "^7.13.10",
    "@babel/core": "^7.13.10",
    "@babel/plugin-proposal-class-properties": "^7.14.5",
    "@babel/preset-env": "^7.13.10",
    "@babel/preset-react": "^7.12.13",
    "@babel/preset-typescript": "^7.13.0",
    "@emotion/core": "^10.0.28",
    "@storybook/addon-a11y": "^6.3.9",
    "@storybook/addon-actions": "^6.3.12",
    "@storybook/addon-essentials": "^6.3.9",
    "@storybook/addon-links": "^6.3.9",
    "@storybook/react": "^6.3.9",
    "@storybook/theming": "^6.3.9",
    "@testing-library/react-hooks": "^7.0.2",
    "@types/enzyme": "^3.10.5",
    "@types/enzyme-adapter-react-16": "^1.0.6",
    "@types/jest": "^26.0.20",
    "@types/lodash": "^4.14.172",
    "@types/react": "^16.14.5",
    "@types/react-dom": "^16.9.5",
    "@types/react-highlight-words": "^0.16.2",
    "@types/react-responsive": "^8.0.2",
    "@types/react-router-dom": "^5.1.7",
    "@types/react-virtualized": "^9.21.11",
    "@types/stopword": "^1.0.2",
    "@types/video.js": "^7.3.26",
    "@typescript-eslint/eslint-plugin": "~2.8",
    "@typescript-eslint/parser": "~2.8",
    "babel-loader": "~8.0",
    "cross-env": "^7.0.2",
    "css-loader": "^5.0.2",
    "dotenv-webpack": "^7.0.3",
    "enzyme": "^3.11.0",
    "enzyme-adapter-react-16": "^1.15.2",
    "enzyme-to-json": "^3.4.4",
    "eslint": "~6.6",
    "eslint-config-prettier": "~6.7",
    "eslint-plugin-react": "^7.19.0",
    "file-loader": "^6.2.0",
    "gh-pages": "^3.1.0",
    "html-webpack-plugin": "^5.3.2",
    "husky": "~3.1",
    "identity-obj-proxy": "^3.0.0",
    "image-webpack-loader": "^7.0.1",
    "jest": "^26.6.3",
    "lint-staged": "~9.4",
    "mini-css-extract-plugin": "^1.3.8",
    "moment": "^2.24.0",
    "npm-run-all": "~4.1",
    "nyc": "^15.1.0",
    "postcss": "^8.3.9",
    "postcss-loader": "^6.1.1",
    "prettier": "^2.0.5",
    "react": "^16.14.0",
    "react-dom": "^16.13.0",
    "rimraf": "~3.0",
    "semantic-ui-css": "^2.4.1",
    "snowpack": "^3.2.2",
    "ts-jest": "^26.1.1",
    "ts-node": "~8.5",
    "typescript": "^3.9.9",
    "webpack": "^5.52.0",
    "webpack-cli": "^4.9.1"
  },
  ```
  
  ```
    "dependencies": {
    "@councildataproject/cdp-design": "^1.0.2",
    "@emotion/react": "^11.4.1",
    "@emotion/styled": "^11.3.0",
    "@material-ui/icons": "^4.11.2",
    "@mozilla-protocol/core": "^13.0.1",
    "@storybook/addons": "^6.3.12",
    "firebase": "^9.0.0-beta.7",
    "lodash": "^4.17.21",
    "moment": "^2.24.0",
    "n-gram": "^2.0.1",
    "react-highlight-words": "^0.17.0",
    "react-localization": "^1.0.17",
    "react-responsive": "^8.2.0",
    "react-router-dom": "^5.2.0",
    "react-virtualized": "^9.22.3",
    "semantic-ui-react": "^2.0.3",
    "stemr": "^1.0.0",
    "stopword": "^1.0.11",
    "video.js": "^7.14.3"
  },
  ```

**Comment by nickmagruder** [comment]
After adding Jackson's small changes (and rerunning npm install) from this PR/issue: https://github.com/CouncilDataProject/cdp-frontend/pull/121/commits/6e582501d37eb45e0199ce587bbe0a29b0aa08d7 / https://github.com/CouncilDataProject/cdp-frontend/pull/121 ```npm run start``` now seems to work and generates a seemingly pared-down version of the frontend, but ```npm run storybook``` still gives the same error as above.

**Comment by nickmagruder** [comment]
No PR yet, but I pushed my changes to my fork (linked below), first I updated these 3 dependencies so "npm install" could complete successfully:
- "@testing-library/react-hooks": "^7.0.2",
- "snowpack": "^3.8.8",
- "webpack-cli": "^4.9.1"

I also ran npm install for the 7 peerDependencies on lines ~123-131. When either trying to run "npm run storybook" or "npm run start" it gave a errors that each of those weren't installed, so I installed each one and it worked afterward (I believe this also applies to line 56: "@storybook/addon-actions". No idea what each of those dependencies do. 

If this is too big or weird of a change to the package.json I can close the branch, and push one that just has the 3 dependency changes from the top.

https://github.com/CouncilDataProject/cdp-frontend/compare/main...nickmagruder:bugfix/fix-node-module-depencies?expand=1
